propagete_reporting_test.go

```diff
@@ -131,7 +132,7 @@ func TestPropagateReporting(t *testing.T) {
 		t.Run(fmt.Sprintf("tc %s", k), func(t *testing.T) {
 			defer tc.workflowsMock.AssertExpectations(t)
 			defer tc.storeMock.AssertExpectations(t)
-			err := propagateReporting(tc.storeMock, tc.workflowsMock, tc.cmdTenant, tc.cmdDryRun)
+			err := propagateReporting(tc.storeMock, tc.workflowsMock, tc.cmdTenant, time.Microsecond, tc.cmdDryRun)
```

Comment:
we could create a new test case, which first calculates the avg time it took to run with throttle eq 0 and then sets the throttle to 2s and verifies that it took on avg longer, and the longer is propotional to the duration.